shared-bindings/bleio/Peripheral.c

```diff
@@ -181,37 +187,55 @@ const mp_obj_property_t bleio_peripheral_name_obj = {
                (mp_obj_t)&mp_const_none_obj },
 };
 
-//|   .. method:: start_advertising(*, connectable=True, data=None)
+//|   .. method:: start_advertising(data, *, scan_response=None, connectable=True, interval=1)
 //|
 //|     Starts advertising the peripheral. The peripheral's name and
 //|     services are included in the advertisement packets.
```

Comment:
Does this replace previous advertising data or do they alternate?

Reply:
The `data` value is not saved between `start_advertising()` calls, so you have to  supply it each time. The `scan_response()` is sent in response to a query for further data from the Central.